src/connectedvmware/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+1.0.2
```

Comment:
Feature added should mark version to increase in minor part, which is 1.1.0

Reply:
Request your favor to solve this issue if possible!

Reply:
Sorry for the delay. updated.